types/index.esm.d.ts

```diff
@@ -527,13 +527,14 @@ export class DatasetController<TElement extends Element = Element, TDatasetEleme
 	protected getMaxOverflow(): boolean | number;
 	draw(): void;
 	reset(): void;
-	getDataset(): ChartDataset;
+  getDataset(): ChartDataset;
```

Comment:
spaces -> tabs

(for every line changed in this file)

Reply:
This entire file appears to be spaces.

Reply:
It looks really inconsistent. The first 379 lines are spaces, the next 900 are tabs, and it keeps switching back and forth

We should probably convert the whole thing to tabs at some point though that seems outside the scope of this PR

Still it might be a good idea to have the new lines you're adding here be tabs because the lines before and after them are tabs and so things don't align properly because of the mismatch

Reply:
Ok, looks like there's a section of tabs in the middle. I had looked at the bottom and it's spaces. Let's do a separate PR to fix that since it will complicate the diff here

Reply:
https://github.com/chartjs/Chart.js/pull/8263 has been merged to standardize the file on tabs, so the lines be added in this PR should be switched to tabs now too